docs/user-guide/src/bmo/detached_annotation.md

```diff
@@ -0,0 +1,38 @@
+# Detached annotation
+
+The detached annotation provides a way to prevent management of a BareMetalHost.
+It works by deleting the host from Ironic without triggering deprovisioning.
+The BareMetal Operator will recreate the host in Ironic again once the annotation is removed.
+
+Normally, deleting a BareMetalHost will always trigger deprovisioning.
+This can be problematic and unnecessary if we just want to, for example, move the BareMetalHost from one cluster to another.
+By applying the annotation before removing the BareMetalHost from the old cluster, we can ensure that the host is not disrupted.
+The next step is then to recreate it in the new cluster without triggering a new inspection.
+See the [status annotation page](./status_annotation.md) for how to do this.
+
+The annotation key is `baremetalhost.metal3.io/detached` and the value can be anything (it is ignored).
```

Comment:
baremetalhost.metal3.io/detached= ?

Reply:
What do you mean? The value can be an empty string for example. If you want to set it though kubectl it would be `kubectl annotate bmh <name> baremetalhost.metal3.io/detached=""`.

Reply:
Got it. Ignore my comment.